src/app/lib/stillingSoekSchema.ts

```diff
@@ -0,0 +1,282 @@
+import { z } from "zod";
+
+import fixLocationName from "@/app/_common/utils/fixLocationName";
+import logger from "@/app/_common/utils/logger";
+import DOMPurify from "isomorphic-dompurify";
+import { addPercentageAtEnd, getAdText, getDate, getExtent, getWorktime } from "@/app/stilling/_data/utils";
+
+export const contactDTOSchema = z.object({
+    id: z.number().optional().nullable(),
+    name: z.string().optional().nullable(),
+    email: z.string().optional().nullable(),
+    phone: z.string().optional().nullable(),
+    role: z.string().optional().nullable(),
+    title: z.string().optional().nullable(),
```

Comment:
Bare mer av nysgjerrighet men trenger disse og f.eks locationSchema å være `nullable()`? Jeg kan ikke huske at vi var så flinke til å sette ting til `null` (men jeg har absolutt ikke full oversikt heller).

Reply:
Det jeg opplevde når jeg klikket meg gjennom alle stillingene lokalt var at enkelte props feilet da forventet verdi var undefined og mottok "null". så la da inn , og noen ganger var de undefined. så virket som om det var litt forskjellig håndtering av det. hadde vært greit å gå over og sjekke apiet for å gjøre ting likt over alt.